webapp/components/admin_console/admin_console.jsx

```diff
@@ -43,9 +43,12 @@ export default class AdminConsole extends React.Component {
 
     render() {
         const config = this.state.config;
+        if (!config) {
```

Comment:
This now fixes the problem while trying to load a page directly with the url address